server/statistics/region_collection.go

```diff
@@ -100,43 +129,47 @@ func (r *RegionStatistics) Observe(region *core.RegionInfo, stores []*core.Store
 		}
 	}
 
-	if len(region.GetPeers()) < desiredReplicas {
-		r.stats[MissPeer][regionID] = region
-		peerTypeIndex |= MissPeer
-	} else if len(region.GetPeers()) > desiredReplicas {
-		r.stats[ExtraPeer][regionID] = region
-		peerTypeIndex |= ExtraPeer
-	}
+	var isOffline bool
```

Comment:
```suggestion
	var hasOffline bool
```

Reply:
IMO, isOffline is more suitable here?

Reply:
The region is offline, which looks like the region is not available. But I think maybe we should replace `offline`?